src/js/state/Tabs/reducer.js

```diff
@@ -119,3 +120,14 @@ function updateTab(state, action: Object) {
 function indexOf(data, id) {
   return data.findIndex((t) => t.id === id)
 }
+
+function reduxAction(action) {
+  return action.type.startsWith("@@")
```

Comment:
I wasn't aware that this is associated with an internal redux event. Would you mind adding a little comment here providing some color on what this is doing, for posterity?